src/docs/devices/KinCony-ADR/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: KinCony-ADR (AC Dimmer)
+date-published: 2024-10-12
+
+type: dimmer
+standard: global
+board: esp32
+---
+
+![Product](esp32-ac-dimmer-adr-1.jpg "Product Image")
+![Product](ADR-lamp.jpg "Product Image")
+![Product](ADR-fan.jpg "Product Image")
+
+## Resources
+
+- [ESP32 pin define details](https://www.kincony.com/forum/showthread.php?tid=5276)
+- [YouTube video tour](https://youtu.be/JMtgOhL1Jb0)
+
+## ESPHome Configuration
+
+Here is an example YAML configuration for the KinCony-ADR board.
+
+```yaml
+esphome:
+  name: adr
+  friendly_name: ADR
+
+esp32:
+  board: esp32dev
+  framework:
+    type: arduino
+
+# Enable logging
+logger:
+
+# Enable Home Assistant API
+api:
+  encryption:
+    key: "ZWeLIQ6pHBi8F7iz8oU50b5e4pOntKh7crgfrRpT0NQ="

```

Comment:
Please remove this value from the file. There should be no passwords or secrets references in any examples. The only exceptions are !secret wifi_ssid and !secret wifi_password.